### PR TITLE
Portfolio desktop

### PR DESCRIPTION
**Pull request by jpol01**
## Adding desktop version

>Objective:  
           * To use media queries to make sure that my website displays correctly on 
            desktop screens.

**The min-width for this project has been set to 1240px the optimal would be 1440px.

Most of the margins, padding and other measurements were provided in percentages, so they could re-adjust easily between the 1240px and the 1440px 

I didn't add the tablet version for this project because was not a requirement.
**

### CODE REVIEW

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @team, 

Good job so far!

✔️ Good commit description
✔️ Desktop version looks great
✔️ Good pull request description

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._
- [ ] The live demo link in your README file is broken, so it would be great if you could fix that or just remove it at once 💯 to make your README file more professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -482,7 +482,7 @@ hr {
 }
 
 /* dont change things above this mark */
-@media (min-width: 992px) {
+@media (min-width: 1240px) {
```
[the remark this answers is not in the document]
Comment (jpol01) [reply style.css]: Fixed both. Thanks

File: index.html [hunk 2]
```diff
@@ -131,22 +132,23 @@ <h3 class="card1-h3">Multi-Post Stories</h3>
...
-          <p class="u-card-p">
+          <p class="u-card-p have2-hidden">
             A daily selection of privately <br />
             personalized reads; no accounts or <br />
             sign-ups required. has been the <br />
             industry's standard
           </p>
 
-          <ul class="u-card-technologies">
+          <ul class="u-card-technologies have2-hidden">
```
[the remark this answers is not in the document]
Comment (jpol01) [reply index.html]: The features to make the orange button the (**See Project button**) appears while the title and the description paragraph vanish was successfully applied. :smile:

File: style.css [hunk 3]
```diff
@@ -482,24 +485,18 @@ hr {
   display: none;
 }
 
-
-
 @media (max-width: 375px) {
```
[the remark this answers is not in the document]
Comment (jpol01) [reply style.css:460]: Fixed thanks.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉✔🍾**
Hello @jpol01 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**